Library/Homebrew/extend/os/linux/keg.rb

```diff
@@ -1,9 +1,8 @@
-# typed: true # rubocop:disable Sorbet/StrictSigil
+# typed: strict
 # frozen_string_literal: true
 
 class Keg
-  undef binary_executable_or_library_files
```

Comment:
Probably want to keep this until #17998 is resolved.

Reply:
I thought the plan was to try to remove and see if we had any test failures. If we need to hold off, I'll close out the Linux PR's since they all have `undef`.

Reply:
Actually, let's just do this now: https://github.com/Homebrew/brew/issues/17998#issuecomment-2290914094